### DISCUSSION TITLE
Troubleshooting Routing Issues in Web Application - Unable to Access Login Page

### DISCUSSION THREAD

**Discussion by rahilansari261** [dubinc/dub#553] [Q&A] [answered]
Whenever I try to access http://localhost:3000/login, I'm automatically redirected to http://localhost:3000/. I initially used port 8888, but it resulted in an undefined error. After switching to port 3000, the homepage worked fine, but I'm unable to navigate to other pages, specifically the login page. I need assistance in resolving this issue and accessing the login page.

**Comment by steven-tey** [comment, marked as answer]
Probably something to do with the app middleware logic: https://github.com/dubinc/dub/blob/main/apps/web/lib/middleware/app.ts
